client/gutenberg/editor/calypsoify-iframe.jsx

```diff
@@ -44,6 +45,8 @@ class CalypsoifyIframe extends Component {
 		postType: PropTypes.string,
 		duplicatePostId: PropTypes.number,
 		pressThis: PropTypes.object,
+		markChanged: PropTypes.func.isRequired,
+		markSaved: PropTypes.func.isRequired,
```

Comment:
With the functions being required, should default props be defined for them?

Reply:
These props are filled by the `protectForm` HOC and [are always defined](https://github.com/Automattic/wp-calypso/blob/7bf6a37f599d3e35dad482cee2cd8cd73dca5466/client/lib/protect-form/index.jsx#L60-L61).